manifests/o/Ookla/Speedtest/CLI/1.2.0/Ookla.Speedtest.CLI.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ookla.Speedtest.CLI
+PackageVersion: 1.2.0
```

```suggestion
PackageVersion: 1.2.0
AppsAndFeaturesEntries:
  - DisplayVersion: 1.2.0.84
```
The package is marketed as `1.2.0` but `speedtest --version` shows `1.2.0.84`, so we'll use AppsAndFeaturesEntries here